CHANGELOG.develop

```diff
@@ -2224,6 +2225,20 @@ Other:
   - Use JSX header without breaking React (thanks to Jam Risser)
   - Fixed React imports when using web beautify. (thanks to Ismael)
   - Made layer depends on node and tern layers (thanks to Sylvain Benner)
+
```

Comment:
Match the other layer entries by removing the empty lines before and after the ReasonML section.